src/matching_transformation.py

```diff
@@ -0,0 +1,22 @@
+from talon import Context, Module
```

Comment:
I'd rename to `matching_pair_symbol`, and then put it into a `modifiers/` directory